osmaxx/api_client/API_client.py

```diff
@@ -4,6 +4,8 @@
 
 logger = logging.getLogger(__name__)
 
+HTTPError = requests.HTTPError
```

Comment:
Because the place(s) where it is used shouldn't have to care that `API_client.HTTPError` is an alias and actually just a `requests.HTTPError`.

The aim is decoupling: `API_client.py` should be the only file dependent on `requests`. Thus if `requests` changes or if we replace `requests`, it should be possible to 'hide' that change from the rest of our system by only changing `API_client.py` accordingly.